packages/contracts-bedrock/src/universal/OptimismMintableERC20Factory.sol

```diff
@@ -104,29 +109,29 @@ contract OptimismMintableERC20Factory is ISemver, Initializable {
     /// @param _name        ERC20 name.
     /// @param _symbol      ERC20 symbol.
     /// @param _decimals    ERC20 decimals
-    /// @return Address of the newly created token.
+    /// @return _localToken Address of the newly created token.
```

Comment:
not sure if there is a convention, but they dont use underscore for localToken in their current version.